Yoast/wordpress-seo
# Replaces WordPress hooks with custom ones for the Configuration Wizard

## Summary

This PR can be summarized in the following changelog entry:

This incidentally also fixes problems with other plugins that rely on a complete WordPress admin environment.
Also improved a line of JavaScript that lead to a new line in the middle of an URL.

Props to @diedexx for helping with this issue.

## Test instructions

This PR can be tested by following these steps:

* Enable Polylang plugin
* Attempt to go through the Configuration Wizard.
* Check if all steps are successfully applied.

Fixes #5928
